template/semantic-ui/layout/admin.html

```diff
@@ -1,6 +1,6 @@
 
 <div class="atk-layout">
-  <div class="ui left vertical menu visible inverted labeled sidebar atk-sidenav">{$LeftMenu}</div>
+  <div class="ui left vertical menu inverted labeled sidebar atk-sidenav">{$LeftMenu}</div>
```

Comment:
This will close the left menu on each page load and this is not the intended behavior.

Reply:
See https://github.com/atk4/ui/blob/3fd37f0bee0d43f01e9eccb41e96b9dc3466d5ca/src/Layout/Admin.php#L111

If you need to use it anywhere else, you have to add the visible class yourself. But if it is hardcoded, it it worng. Ok to merge?

Reply:
The  make it open by default ;) not close by default

Reply:
But then it can flash... I am just fixing what was broken by PR #1112. Is there any issue with this fix?

Reply:
> This will close the left menu on each page load and this is not the intended behavior.

Reply:
That is what the option `isMenuLeftVisible` is for and it worked perfectly, you just have to solve the hardcoded visibility in the template - this is solved by this PR and check, make the `isMenuLeftVisible` working again. Btw. `isMenuLeftVisible` is set to true by default, so your wanted behaviour will be matched.

Reply:
then set it to true by default ;-)

Reply:
https://github.com/atk4/ui/blob/3fd37f0bee0d43f01e9eccb41e96b9dc3466d5ca/src/Layout/Admin.php#L42 !

What need to be done next:
- fix https://github.com/atk4/ui/blob/3fd37f0bee0d43f01e9eccb41e96b9dc3466d5ca/src/Layout/Admin.php#L111 probably by moving it to init() here https://github.com/atk4/ui/blob/3fd37f0bee0d43f01e9eccb41e96b9dc3466d5ca/src/Layout/Admin.php#L57 and do the same what the 1st click of the visibility button does
- commit it on top of this PR
- merge it

Reply:
and why not doing it in this PR then?

Reply:
I have not resource to fixing these...

Reply:
This PR will break the intended behavior, either submit a fully working PR or just submit an issue and I will check when I have some time.

Reply:
Please help with this line https://github.com/atk4/ui/pull/1157/files#diff-5bb52268570a74fcdec90d6f8b8f3ab9R71 , I do not know the right Atk way how to add the visibility class.

Reply:
Leave it to me. The template needs to be changed. I will do it when I have a chance.